src/client/Microsoft.Identity.Client/Cache/CacheSessionManager.cs

```diff
@@ -113,7 +113,7 @@ private async Task RefreshCacheForReadOperationsAsync(CacheEvent.TokenTypes cach
                                 {
                                     var args = new TokenCacheNotificationArgs(
                                        TokenCacheInternal,
-                                       _requestParams.ClientId,
+                                       _requestParams.AppConfig.ClientId,
```

Comment:
There's no point in copying things like ClientId from the app config to the request params. Client ID is immutable.

Reply:
agree.

---
In reply to: [566259871](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2370#discussion_r566259871) [](ancestors = 566259871)